.github/workflows/release.yaml

```diff
@@ -16,6 +16,8 @@ name: release
 
 on:
   workflow_dispatch: # testing only, trigger manually to test it works
+  schedule:
+    - cron: 0 0 * * * # every day at 0:00, build latest on default branch (main)
```

Comment:
nit: can we change this to have a bit of randomness since its likely a lot of people will be doing things at 0:00, maybe 0 23 or something?